go.mod

```diff
@@ -3,15 +3,16 @@ module github.com/bazelbuild/bazel-gazelle
 go 1.20
 
 require (
-	github.com/bazelbuild/buildtools v0.0.0-20230510134650-37bd1811516d
-	github.com/bazelbuild/rules_go v0.39.1
+	github.com/bazelbuild/buildtools v0.0.0-20230823134145-56628089db01
+	github.com/bazelbuild/rules_go v0.41.0
 	github.com/bmatcuk/doublestar/v4 v4.6.0
 	github.com/fsnotify/fsnotify v1.6.0
 	github.com/google/go-cmp v0.5.9
 	github.com/pmezard/go-difflib v1.0.0
 	golang.org/x/mod v0.12.0
-	golang.org/x/sync v0.2.0
-	golang.org/x/tools v0.9.1
+	golang.org/x/sync v0.3.0
+	golang.org/x/tools v0.12.0
```

Comment:
I'm surprised that updating golang.org/x/tools here does not trigger [this bug in rules_go](https://github.com/bazelbuild/rules_go/issues/3640), since  that one expects a different function signature for a function in tools.
But if CI is green i shall not complain 😅

Reply:
Interesting. If we think this is an issue we could wait on this upgrade.